spk/rutorrent/Makefile

```diff
@@ -44,6 +44,8 @@ ENV += BUSYBOX_CONFIG="$(BUSYBOX_CONFIG)"
 
 include ../../mk/spksrc.spk.mk
 
+SPK_DEPENDS = "python3>=3.7.7:php5.6"
```

Comment:
Shouldn't we, instead, be checking for the presence of python before starting / installing and disabling the plugin if missing ?

Reply:
I thought it was much more simple to add `SPK_DEPENDS` which forces the installation of prerequisites packages in the first place, thus ensuring the plugin will work?  This way, in theory, there is no need to check anything and we can assume the plugin to work.